core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -106,10 +107,11 @@ public void add(int size) {
     }
 
     public void print(String indent) {
-      System.out.println(indent + "Up to size      count      %-age");
+      System.out.println(indent + "Up to size      Count      %-age");
       for (int i = 1; i < countBuckets.length; i++) {
-        System.out.println(String.format("%s%11.0f : %10d %6.2f%%", indent, Math.pow(10, i),
-            countBuckets[i], sizeBuckets[i] * 100. / totalSize));
+        System.out.println("%s%11s : %10d %6.2f%%", indent,
```

Comment:
```suggestion
        System.out.printf("%s%11s : %10d %6.2f%%", indent,
```

That is my mistake, I forgot to add this change.